pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -94,6 +94,9 @@
   hoc_overview_csc_header: 'Introducing CS Connections'
   hoc_overview_csc_desc: 'Our newest curriculum highlights the connections between computer science and other subjects like math, language arts, science, and social studies. Through these modules, projects, and lessons, classrooms can explore their usual subjects in exciting new ways!'
   hoc_overview_csc_learn_more: 'Learn more about CS Connections'
+  hoc_danceparty_alt_text: 'gif of characters dancing underneath the text, Code your own dance party'
```

Comment:
How about "Animated characters dancing to music underneath the text 'Code your own dance party'"

Reply:
(following a best practice of not starting alt-text with "picture of..." or "image of..."

Reply:
thanks for the suggestions! I'll update